src/components/marker-chart/Canvas.js

```diff
@@ -295,7 +295,7 @@ class MarkerChartCanvas extends React.PureComponent<Props, State> {
     }
   }
 
-  hitTest = (x: CssPixels, y: CssPixels): IndexIntoMarkerTiming | null => {
+  hitTest = (x: CssPixels, y: CssPixels): IndexIntoMarkers | null => {
```

Comment:
This was a lie as well, and as a result all the other types in the same file were lies too.